internal_test_data/multi_region_dc_test_case.yaml

```diff
@@ -1,4 +1,4 @@
-stress_cmd: "cassandra-stress write cl=QUORUM n=1200300 -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=2,us-west-2scylla_node_west=1)' -mode cql3 native -rate threads=200 -pop seq=400200300..600200300"
+stress_cmd: "cassandra-stress write cl=LOCAL_QUORUM n=1200300 -schema 'replication(strategy=NetworkTopologyStrategy,us-eastscylla_node_east=2,us-west-2scylla_node_west=1)' -mode cql3 native -rate threads=200 -pop seq=400200300..600200300"
```

Comment:
Just `QUORUM` will make us get `2` answers for any query in any case.
The `LOCAL_QUORUM` may provide only `1` answer in case of the region with one node.
And usage of the latter one here sounds wrong to me.

Reply:
Using cl not `LOCAL_` or `EACH_` will creates more load than we expect and ignore DCs configuration.
In multiDc environment we create keyspace with replication class "NetworkTopologyStrategy" and RF is configured as, for example:
```
replication = {'class': 'NetworkTopologyStrategy', 'eu-westscylla_node_west': 3, 'us-west-2scylla_node_west': 2, 'us-eastscylla_node_east': 1}
```

It will expect answer from all DCs but does not run duplicated load.
We do not need to run "NetworkTopologyStrategy" as simple. It's not correct

Reply:
Anyhow this is just configuration for unittest, it's not really critical to change it.